mlflow/store/cli.py

```diff
@@ -0,0 +1,101 @@
+from mlflow.utils.logging_utils import eprint
+
+import click
+
+from mlflow.tracking import _get_store
+from mlflow.store.artifact_repo import ArtifactRepository
+from mlflow.utils.proto_json_utils import message_to_json
+
+
+@click.group("artifacts")
+def commands():
+    """Upload, list, and download artifacts from an MLflow artifact repository."""
+    pass
+
+
+@commands.command("log-artifact",
+                  help="Logs a local file as an artifact of a run, optionally within a " +
+                       "run-specific artifact path. Run artifacts can be organized into " +
+                       "directories, so you can place the artifact in a directory this way.")
+@click.option("--local-file", "-l", required=True,
+              help="Local path to artifact to log")
+@click.option("--run-id", "-r", required=True,
+              help="Run ID into which we should log the artifact.")
+@click.option("--artifact-path", "-a",
+              help="If specified, we will log the artifact into this subdirectory of the " +
+                   "run's artifact directory.")
+def log_artifact(local_file, run_id, artifact_path):
+    """
+    Creates a new experiment in the configured tracking server.
```

Comment:
Wrong doc strings? FYI you can also put the command help here instead of adding a `help=` at the top.